src/bundles/files.js

```diff
@@ -243,15 +296,17 @@ export default {
       }
 
       for (const file of ipfsFiles) {
+        const fileId = shortid.generate()
```

Comment:
Why not use the hash as the ID instead of a shortid?

Reply:
I could, but I prefer to keep the logic consistent with the add files one (when adding files I'm using shortId because I only get the hash after the file is added).